src/databases/unv/avtunvFileFormat.C

```diff
@@ -3874,6 +3884,816 @@ avtunvFileFormat::ReadFile()
+                                                ptr += nchar ;
+                                                if (ier != -1)
+                                                {
+                                                    nbletsptyp[ier]++;
+#if !defined(MDSERVER)
+                                                    // Reads the nodes as much as we need, not as we can
+                                                    for (int inod=0; inod < nbnel; inod++)
+                                                    {
+                                                        int temp;
+                                                        if (sscanf(ptr, "%d %n", &temp, &nchar) != 1)
+                                                        {
+                                                            // error: not a number
+                                                            break;
+                                                        }
+                                                        nodes[inod] = temp ;
+#if INTERACTIVEREAD
+                                                        if (debuglevel >= 5) fprintf(stdout, "nod(%d)=%d\n",inod,temp);
+#endif                                                        
+                                                        ptr += nchar ;
+                                                    }
+                                                    anUnvElement.label = label;
+                                                    anUnvElement.typelt = itypelt;
+                                                    anUnvElement.matid = numat;
+                                                    anUnvElement.nodes = new int[nbnel];
+                                                    for (int i=0; i < nbnel; i++)
+                                                        anUnvElement.nodes[i] = nodes[i];
+                                                    
+#if INTERACTIVEREAD
+                                                    if (debuglevel >= 4) fprintf(stdout, "\telt %d, #nodes=%d, first/last node=(%d,%d)\n",
+                                                                                 anUnvElement.label,nbnel,anUnvElement.nodes[0],anUnvElement.nodes[nbnel-1]);
+#else
+                                                    debug4 << "\telt " << anUnvElement.label << ", #nodes=" << nbnel << ", first/last node=(" << anUnvElement.nodes[0] << "," << anUnvElement.nodes[nbnel-1] << ")" << endl;
+#endif
+                                                    meshUnvElements.insert(anUnvElement);
+#endif
+                                                    anUnvElement.number++;
+                                                }
+                                            }
+                                            number+=lbelts ;
+#if INTERACTIVEREAD
+                                            if (debuglevel >= 4) fprintf(stdout, "Elements Counter=%d / %d\n",number, nbelts) ;
```

Comment:
I assume you want an "#else" case for debug4.

Reply:
I had made the corrections in the wrong file...
Now it should be OK